apps/bfd-pipeline/bfd-pipeline-rda-grpc/pom.xml

```diff
@@ -205,11 +205,6 @@
             <artifactId>aws-crt</artifactId>
             <version>${aws.crt.version}</version>
         </dependency>
-        <dependency>
-            <groupId>jakarta.persistence</groupId>
-            <artifactId>jakarta.persistence-api</artifactId>
-            <version>3.2.0-M2</version>
-        </dependency>
```

Comment:
We ignore milestone versions for the purposes of the [maven-versions plugin](https://www.mojohaus.org/versions/versions-maven-plugin/index.html) under `apps/rules.xml`, however, that only applies when running the plugin explicitly, e.g. `mvn versions:display-dependency-updates`. This mismatch has been throwing errors which I only noticed with a verbose maven output when diagnosing the issues here:
> [WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: jakarta.persistence:jakarta.persistence-api:jar -> version ${jakarta.persistence-api.version} vs 3.2.0-M2 @ line 208, column 21

Changes to `apps/bfd-pipeline/bfd-pipeline-rda-grpc/pom.xml` and `apps/pom.xml` are merely targets of opportunity here.